velox/docs/configs.rst

```diff
@@ -109,6 +109,11 @@ Expression Evaluation Configuration
      - bool
      - false
      - This flags forces the cast from float/double to integer to be performed by truncating the decimal part instead of rounding.
+   * - cast_string_to_date_non_standard
+     - bool
+     - false
+     - This flags allows the cast from string to date accept patterns other than the "[+-](YYYY-MM-DD)" format.
+       Valid patterns include (YYYY, YYYY-MM, YYYY-MM-DD), and any patterns prefixed with [+-]".
```

Comment:
should we also mention other cases like
Digits after trailing spaces: "YYYY-MM-DD 123"
Trailing character 'T': "YYYY-MM-DDT"
Digits after 'T': "YYYY-MM-DDT123123"